src/cascadia/TerminalControl/TermControl.idl

```diff
@@ -40,6 +42,7 @@ namespace Microsoft.Terminal.TerminalControl
         Boolean ShouldCloseOnExit { get; };
         Windows.Foundation.Size CharacterDimensions { get; };
         Windows.Foundation.Size MinimumSize { get; };
+        Single SnapDimensionToGrid(Boolean widthOrHeight, Single dimension);

```

Comment:
Because in `Pane` we operate on floats. But really its only matter of where the conversion form doubles to floats is done, so it probably has sense to do it in `Pane` and have doubles in this interface. On the other hand this would be inconsistent with previous two properties of type `Windows.Foundation.Size` which has float values.